packages/teams-js/test/utils.ts

```diff
@@ -130,9 +130,22 @@ export class Utils {
   };
 
   public findMessageByFunc = (func: string): MessageRequest | null => {
```

Comment:
FindMessageByFunc and FindKthMessageByFunc could be combined to one by setting k: number as an optional parameter

Reply:
Updated with doc.